### PR TITLE
Added in and not_in filters to IntegerField

### PR DESCRIPTION
**Pull request by LiamAttClarke**
Related issue: https://github.com/keystonejs/keystone/issues/1958

### CODE REVIEW

**Comment by gautamsi** [comment]
I think the contributor specs are taken care by a bot, you should not have to add your detail in there.  also deleting `CONTRIBUTING.md` is not necessary for this PR.

File: packages/fields/src/types/Integer/views/Controller.js [hunk 1]
```diff
@@ -1,21 +1,26 @@
...
+    let arg = value.replace(/\s/g, '');
+    if (['in', 'not_in'].includes(type)) {
+      arg = `[${arg}]`;
+    }
+    return `${key}: ${arg}`;
   };
-  getFilterLabel = ({ label }: FilterLabel): string => {
-    return `${this.label} ${label.toLowerCase()}`;
+  getFilterLabel = ({ label, type }: FilterLabel): string => {
+    const suffix = ['in', 'not_in'].includes(type) ? ' (comma separated)' : '';
```
Comment (timleslie) [inline packages/fields/src/types/Integer/views/Controller.js]: I think it's probably easier if we just put this `suffix` explicitly on the labels at line 70 and 75, it'll keep this code nice and simple.

Comment (LiamAttClarke) [reply packages/fields/src/types/Integer/views/Controller.js]: I agree, that would be better. I was trying to keep the code consistent with the UUID and MongoID fields.

Comment (LiamAttClarke) [reply packages/fields/src/types/Integer/views/Controller.js]: Looks like adding '(comma separated)' directly on the field label results in a UI where the suffix is added to the filter drop-down and the active filter pill interface. It seems like the term `label` is overloaded for this context.

For this bug fix, I suggest we leave this implementation as it's consistent with other fields. I suggest refactoring fields to have an option to override the input placeholder value.

The FieldController.getFilterTypes could return a list of types with the included placeholder:
```
getFilterTypes = () => [
    {
      type: 'in',
      label: 'Is one of',
      placeholder: 'Is one of (comma separated)',
      getInitialValue: () => '',
    }
]
```

**Pull request merged**